src/base.css

```diff
@@ -1344,6 +1346,21 @@ Possible extra rowspan handling
 		font-weight: bold;
 	}
 
+/** Character examples (from the I18N template) *******************************/
+.codepoint bdi {
+    line-height: 1em;
+    font-size: 120%;
+    margin-inline: 0.25em;
+}
+
+.uname {
+    font-family: Arial, monospace;
+	font-size: 85%;
```

Comment:
Didn't you want `font-variant: all-small-caps` here?

Reply:
It's not quite the same effect: https://www.w3.org/TR/css-fonts-4/#font-variant-caps-prop

> would mask bugs in the user's input of the Unicode character name

Is casing really a likely error? I imagine the most likely error is that it'll be written out in all lowercase or in title caps, both of which wouldn't look terribly wrong either way. And it doesn't seem like it would cause a meaningful loss in information, the uppercasing of these names is basically just a convention, isn't it?

Reply:
This is super-helpful. I made the change, removing the font-family in favor of `all-small-caps`. When I did that I noticed that the digits in a code point (ASCII decimal digits 0-9) display badly compared to small caps for hex digits A-F, so I also added a `font-variant-numeric` with value `oldstyle-nums`. @r12a: have another look at this and see what you think.